sdk/message/src/sanitized.rs

```diff
@@ -323,20 +326,23 @@ impl SanitizedMessage {
             })
     }
 
+    #[cfg(feature = "bincode")]
```

Comment:
These feature annotations pretty much give the full roadmap about how we can remove bincode from everything -- it's going to be really useful in the future :pray: